manifest_template.yml

```diff
@@ -160,6 +162,10 @@ apigee:
 {% if ENV.name == 'internal-qa' %}
           - identity-service-{{ ENV.name }}-int
           - identity-service-mock-{{ ENV.name }}
+{% endif %}
```

Comment:
Github wont let me comment on the line, but I'm thinking as with the INTERNAL_DEV_VARIANTS we should have the VARIANT name in the Apigee artefacts (product, proxy, catalog). 

Otherwise the dep proxy would just be the generic dev named proxy/products/etc.

I'm not sure if there is a way of doing this without making the manifest_template even more complicated!

Maybe we could collapse the internal-dev and dev sections (and do a bit more templating). Although that would widen the scope of testing/impact.

We do only have dep variant for dev currently so there may be a simpler way, although that could turn out even less readable.

Maybe have a thing and then you/Jack/me could have a chat.

Reply:
I agree yup, it would be nice if we could collapse the new APIM dev environments into the existing internal-dev for loop somehow, though to do this we'd need to refactor how we associate the identity service proxies to our created products. Perhaps some sort of map or object structure allowing you to explicitly declare the variants for a given environment and the other proxies each environment should be associated with. Implementing this would be a major change to this file though.